api/voice-react-native-sdk.api.md

```diff
@@ -255,19 +251,19 @@ export namespace Call {
         export type Ringing = () => void;
     }
     export enum QualityWarning {
-        ConstantAudioInputLevel = "constant-audio-input-level",
-        HighJitter = "high-jitter",
-        HighPacketLoss = "high-packet-loss",
-        HighRtt = "high-rtt",
-        LowMos = "low-mos"
+        'ConstantAudioInputLevel' = "constant-audio-input-level",
+        'HighJitter' = "high-jitter",
+        'HighPacketLoss' = "high-packet-loss",
+        'HighRtt' = "high-rtt",
+        'LowMos' = "low-mos"
     }
     export enum Score {
-        Five = 5,
-        Four = 4,
-        NotReported = 0,
-        One = 1,
-        Three = 3,
-        Two = 2
+        'Five' = "five",
+        'Four' = "four",
+        'NotReported' = "not-reported",
+        'One' = "one",
+        'Three' = "three",
+        'Two' = "two"

```

Comment:
This enum is public right?

Reply:
Yes, that's correct. This is a public enum. Are you concerned about this being a breaking change?